features/include_conversion.feature

```diff
@@ -0,0 +1,5 @@
+Feature: Include Conversion
+
+  Scenario: Convert includes
+    Then I test the file "includes" with filter "dummy_includes/transform.*" and prefix "transformed."
```

Comment:
Should the prefix include a dot? I would prefer that is handled automatically by dstep.